mailer.js

```diff
@@ -198,9 +219,11 @@ module.exports = function (log) {
     if (message.redirectTo) { query.redirectTo = message.redirectTo }
     if (message.resume) { query.resume = message.resume }
 
-    var link = this.verificationUrl + '?' + qs.stringify(query)
+    var link = this._generateUTMLink(this.verificationUrl, query, templateName, 'activate')
```

Comment:
It seems to me the pattern of generating 2/3/4/5 links in sequence could be collapsed into a helper function that generates the primary link, an alternative link, a one click link, and perhaps the support links. The function would return an object with several properties, `link`, `alternative, `oneClick`, `support`, etc.